oneapp-server/types/Application.test.js

```diff
@@ -0,0 +1,72 @@
+const { createTestClient, dataSources, services } = require('../__utils/TestingUtils');
+const client = createTestClient();
+
```

Comment:
Let's rebase atop the new Application.test.js file introduced in #440

Reply:
Agreed. I want to hold this PR until all of yours are in, because I also need to do an end-to-end test against the whole application submission flow.